CHANGELOG.rst

```diff
@@ -2,14 +2,19 @@
 Changelog
 =========
 
+Version 0.7.23
+=============
+Modified:
+
+- Added scaling factors for tuning HTP and VTP areas (#49)
```

Comment:
Just for sake of consistency, but honestly, I wouldn't have mentioned it if it had been the only point.
```suggestion
- Added scaling factors for tuning HTP and VTP areas. (#49)
```